### ISSUE TITLE
Ingame 'load game' button does nothing

### ISSUE THREAD

**Issue by RM87** [Return-To-The-Roots/s25client#203]
Not jet coded?

**Comment by Flamefire** [comment]
The behaviour of that button should be specified. In the original you just load a new game. RTTR is Multiplayer centric. So you can't just load a new game. One possible option is to load the game and switch to the "start game" screen where you can wait for other players. However the current game will also be closed which will affect the other players, so what to do in that case? Add a "Are you sure?" dialog and then just terminate the game?

**Comment by Spikeone** [comment]
What about hiding that button in multiplayer sessions?

In replay sessions it could jump to the replay selection screen and terminate the replay if an other is chosen.

In single player this could jump to the load screen already available. I think as sling as on the fly join and map changes are not working, such functions are useless for mp purposes

**Comment by Flamefire** [comment]
That button is now hidden. I updated it as we could bring it back as simply showing the load game window and switch to the game screen when one is selected. However: What type of game should be started? Probably same as currently running one?

**Comment by Spikeone** [comment]
Does this also work for multiplayer as you have to check if free slots >= current used slots. Also I think double click shouldn't work or an extra window asking if you really want to stop the current game should appear.

**Comment by Flamefire** [comment]
Yes. It won't start the new game directly but proceed to the lobby screen.
I'd allow double-click. You already need to open the window AND select a game via double-click. That is enough safety w/o annoying the user with a "Do you really want me to do what you told me to do"-Dialog ala windows.

**Comment by Spikeone** [comment]
Didn't think about that (is there a load hotkey?) but I guess you are right - maybe autosave on leave just in case.
